src/main/resources/templates/db_scripts/schema.json

```diff
@@ -56,6 +56,36 @@
           "tOps": "ADD",
           "caseSensitive": true,
           "removeAccents": false
+        },
+        {
+          "fieldName": "checkinServicePointId",
```

Comment:
Could you please add `fromModuleVersion`?

Reply:
Done.